src/store/action.js

```diff
@@ -11,5 +15,21 @@ export const ActionCreator = {
   setOffers: (offers) => ({
     type: ActionType.SET_OFFERS,
     payload: offers
+  }),
+  requireAuthorization: (status) => ({
+    type: ActionType.REQUIRED_AUTHORIZATION,
```

Comment:
Может быть тут больше подойдёт
```suggestion
    type: ActionType.SET_AUTHORIZATION_STATUS,
```
?